Tests/ASCollectionViewThrashTests.mm

```diff
@@ -0,0 +1,184 @@
+//
+//  ASTableViewThrashTests.mm
+//  Texture
+//
+//  Copyright (c) Facebook, Inc. and its affiliates.  All rights reserved.
+//  Changes after 4/13/2017 are: Copyright (c) Pinterest, Inc.  All rights reserved.
+//  Licensed under Apache 2.0: http://www.apache.org/licenses/LICENSE-2.0
+//
+
+#import <XCTest/XCTest.h>
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+#import <AsyncDisplayKit/ASCollectionView.h>
+#import <stdatomic.h>
+
+#import "ASThrashUtility.h"
+
+@interface ASCollectionViewThrashTests : XCTestCase
+
+@end
+
+@implementation ASCollectionViewThrashTests  {
```

Comment:
Per [our coding convention](https://github.com/TextureGroup/Texture/blob/master/CONTRIBUTING.md), brackets should be on the following line (and ASTableViewThrashTests.mm doesn't adhere to it). Can you update the rest of this PR to follow this rule?